Description: Fix header path to get rid of compile errors
Author: Umang Parmar <umangjparmar@gmail.com>
Forwarded: no
Last-Update: 2018-06-13

--- a/libsepol/src/private.h
+++ b/libsepol/src/private.h
@@ -14,7 +14,7 @@
 #endif
 
 #include <errno.h>
-#include <dso.h>
+#include "dso.h"
 
 #ifdef __APPLE__
 #define __BYTE_ORDER  BYTE_ORDER
--- a/libsepol/src/util.c
+++ b/libsepol/src/util.c
@@ -27,7 +27,7 @@
 #include <sepol/policydb/flask_types.h>
 #include <sepol/policydb/policydb.h>
 #include <sepol/policydb/util.h>
-#include <dso.h>
+#include "dso.h"
 
 struct val_to_name {
 	unsigned int val;
